runtime/wasm/src/host.rs

```diff
@@ -26,18 +24,12 @@ lazy_static! {
         .map(Duration::from_secs);
     static ref ALLOW_NON_DETERMINISTIC_IPFS: bool =
         std::env::var("GRAPH_ALLOW_NON_DETERMINISTIC_IPFS").is_ok();
-    static ref ALLOW_NON_DETERMINISTIC_3BOX: bool =
-        std::env::var("GRAPH_ALLOW_NON_DETERMINISTIC_3BOX").is_ok();
-    static ref ALLOW_NON_DETERMINISTIC_ARWEAVE: bool =
-        std::env::var("GRAPH_ALLOW_NON_DETERMINISTIC_ARWEAVE").is_ok();
```

Comment:
Shouldn't we keep this feature flags and check them alongside the `apiVersion` <= `0.0.4`?

Reply:
Maybe if this were widely used it would be nice to catch that, but it's not necessary.